### ISSUE TITLE
Prepare manuals for 0.1.0 release

### ISSUE THREAD

**Issue by jesperpedersen** [pgmoneta/pgmoneta_ext#20]
Go through the user and developer manuals and assume that the extension is installed on yum.postgresql.org, and filter information based on what a user should know, and what a developer should know.

Likely a lot can be "aligned" with the manuals from pgmoneta/core

**Comment by jesperpedersen** [comment]
@GuChad369 Can you take this ?

**Comment by GuChad369** [comment]
Yes, but could you tell me what manual you are looking for? We already have a manual that we are developing.

**Comment by jesperpedersen** [comment]
Look at the core manuals - the extension manuals should read in the same way, so people can install from yum.postgresql.org f.ex. We need the postgresql.conf part, the restart part and so on.

Remove the extension from your PostgreSQL installation, and then work through it while updating the manuals

**Comment by GuChad369** [comment]
Is `pgmoneta_ext` available on yum.postgresql.org? I'm unable to install it through yum.

``` console
No match for argument: pgmoneta_ext
Error: Unable to find a match: pgmoneta_ext
```

**Comment by jesperpedersen** [comment]
Not yet - we need the tag first. But, you can assume it will be available

**Comment by jesperpedersen** [comment]
Just copy'n'paste with pgmoneta_ext with all the instruction - well, include pgmoneta as well...
